docker/monorail/config.json

```diff
@@ -29,6 +29,14 @@
           "proxiesEnabled": true,
           "authEnabled": false,
           "routers": "southbound-api-router"
+      },
+	  {
+          "address": "0.0.0.0",
```

Comment:
In our old discussion, the IP for southbound is limited to 172.31.128.1 which communicate with nodes, and northbound could be from all NICs. I found https://github.com/RackHD/RackHD/blob/master/packer/ansible/roles/monorail/files/config.json also use '0.0.0.0' for 9988 port, could you also help update to '172.31.128.1' there.